en/install-maas.md

```diff
@@ -0,0 +1,308 @@
+admin user has been created yet.
+
+A MAAS admin account is needed before we can start configuring MAAS. This
+needs to be done on the command line by typing the following:
+
+```bash
+sudo maas createadmin
+```
+You'll be asked for a username, a password and an email address. The following
+text will assume `admin` was used as the username. 
+
+!!! Note: 
+    MAAS does not currently make use of the email address. 
+
+A final question will ask whether you want to import SSH keys. MAAS uses the
+public SSH key of a user to manage and secure access to deployed nodes, just as
+you might with managed servers or remote machines. Press `Enter` to skip this
+as we'll do this from the web UI in the next step.
+
+## On-boarding
+
+Now we've created an admin account, the web interface will update to ask for
+login credentials. With credentials successfully accepted, the web interface
+will launch the 'Welcome to MAAS' on-boarding page:
+
+![welcome to maas][install-maas_welcome]
+
+This is the first page of two that will step through the final steps necessary
+for MAAS to get up and running. Unless you have specific requirements, most of
+these options can be left at their default values:
+
+- **Connectivity**: important services that default to being outside of your
+  network. These include package archives and the DNS forwarder.
+
+- **Ubuntu**: this section refers to the versions and architectures of the
+  Ubuntu images MAAS will import and use on deployed nodes. Select `14.04 LTS`
+  alongside `16.04 LTS` as an add an additional image. 
+
+![Ubuntu images][install-maas_images]
+
+- **Keys**: You can conveniently import your public SSH key(s) from both
+  Launchpad and Github by entering your user id for these services. To add a
+  local public key file, usually `HOME/ssh/id_rsa.pub`, select `Upload` and paste
+  file contents into the box that appears. Click `Import` to fix the setting.
+
+![SSH key import][install-maas_sshkeys]
+
+If you need to generate a local SSH public/private key pair, type `ssh-keygen -t rsa` 
+from the Linux account you'll control MAAS from, and when asked, leave
+the passphrase blank.
+
+Adding SSH keys completes this initial MAAS configuration. Click `Go to the dashboard` 
+to move to the MAAS dashboard and the device discovery process
+
+## Networking
+
+By default, MAAS will monitor local network traffic and report any devices it
+discovers on the 'Device discovery' page of the web UI. This page also
+functions as the landing page for the dashboard and will be the first you see
+progressing from the installation on-boarding.
+
+![Device discovery][install-maas_discovery]
+
+Before taking the configuration further, we need to tell MAAS about our network
+and how we'd like connections to be configured. 
+
+These options are managed from the `Subnets` page of the web UI. The subnets
+page defaults to listing connections by fabric and MAAS creates one fabric per
```

Comment:
I have currently linked to the 'Concepts' page of the MAAS docs. But I think you're right. As we're attempting to make this as self-contained as possible, it may make sense to include an overview of the main terms in the intro.